packages/dds/tree/src/forest/types/index.ts

```diff
@@ -0,0 +1,10 @@
+/*!
+* Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+* Licensed under the MIT License.
+*/
+
+export {
```

Comment:
I don't think the "types" folders are helpful. I think grouping things logically would make more sense.

If you want to have nested folders in here, having a 'reader' folder could make sense: its a unit that can stand on its own and we might someday want to move out of forest.

"types" on the other hand just encourages us to separate functions from types, which makes the choice of method vs free function impact what file things go in, which seems bad.

Reply:
Okay, I've removed the /types/ folders.